Kickstarter-iOS/Views/Controllers/ManagePledgeViewController.swift

```diff
@@ -44,15 +46,6 @@ final class ManagePledgeViewController: UIViewController {
       |> \.translatesAutoresizingMaskIntoConstraints .~ false
   }()
 
-  private let viewModel: ManagePledgeViewModelType = ManagePledgeViewModel()
```

Comment:
We've mostly used two step configuration
1. Instantiate
2. Configure

So I've gotten rid of this because it was effectively introducing pattern we've been trying to move away from and making the `configureWith` function obsolete.